src/infi/clickhouse_orm/database.py

```diff
@@ -75,16 +75,16 @@ def create_table(self, model_class):
         Creates a table for the given model class, if it does not exist already.
         '''
         # TODO check that model has an engine
-        if model_class.readonly:
-            raise DatabaseException("You can't create read only table")
+        if model_class.system:
```

Comment:
Can you please explain why you stopped checking if the model is readonly?

Reply:
I've divided readonly flag into 2 flags: readonly and system. Before this, readonly flag restricted dropping and creating tables (it was used in system models, and it was suitable for them). But now I've added Merge engine. Table with this engine can be created or dropped, but INSERTs are restricted. So I needed a mechanism to control these restrictions separatly: now readonly flag controls INSERTs only, and system flag controls both INSERTs and CREATE/DROP TABLE.